stdlib/jdbc/src/main/ballerina/src/java.jdbc/client.bal

```diff
@@ -78,18 +80,21 @@ public type Client client object {
     #           may or may not continue to process the remaining commands in the batch. This property can be
     #           used to override this behavior. When it is set to true, if there is a failure in a few commands and
     #           the JDBC driver continues with the remaining commands, the successfully executed commands in the batch
-    #           also will get rolled back.
+    #           also will get rolled back
+    # + getGeneratedKeys - Allow to retrieves auto-generated keys after a statement has been executed if the value is
+    #                      `true`. If it is not supported by the used JDBC driver, still it will return `nil`.
```

Comment:
# + getGeneratedKeys - Indicate that generated keys should be made available for retrieval. If the value is set to true, auto-generated keys are made available to retrieve when the statement is executed. If the underline JDBC driver does not support it, still it will return `()` value.